app/src/ui/app.tsx

```diff
@@ -494,6 +497,18 @@ export class App extends React.Component<IAppProps, IAppState> {
     }
   }
 
+  private testShowNotification() {
+    if (!__DEV__) {
+      return
+    }
+
+    showNotification(
+      'Test notification',
+      'Click here! This is a test notification',
+      () => this.props.dispatcher.showPopup({ type: PopupType.About })
```

Comment:
It's showing the about dialog because it's the easiest one and I didn't wanna create another dialog just for this 🤣 

I also didn't want to use `alert` because I wanted to make sure that clicking on the notification focused on the app's window.